explorer.schema.json

```diff
@@ -0,0 +1,723 @@
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "name",
+                "value"
+            ]
+        },
+        "distinct.values::integer": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "const": "distinct.values"
+                },
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "value": {
+                                "oneOf": [
+                                    {
+                                        "type": "integer"
+                                    },
+                                    {
+                                        "const": "--OTHER--"
+                                    }
+                                ]
+                            },
+                            "count": {
+                                "type": "integer"
+                            }
+                        }
+                    }
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "name",
+                "value"
+            ]
+        },
+        "columnMetrics": {
+            "type": "object",
+            "properties": {
+                "column": {
+                    "type": "string"
+                },
+                "columnType": {
+                    "type": "string",
+                    "enum": [
+                        "integer",
+                        "real",
+                        "text",
+                        "timestamp",
+                        "date",
+                        "datetime",
+                        "boolean",
+                        "unknown"
+                    ]
+                },
+                "status": {
+                    "type": "string",
+                    "enum": [
+                        "New",
+                        "Validating",
+                        "Processing",
+                        "Complete",
+                        "Canceled",
+                        "Error"
+                    ]
+                },
+                "metrics": {
+                    "type": "array"
+                }
+            },
+            "additionalProperties": false,
+            "required": [
+                "column",
+                "columnType",
+                "status",
+                "metrics"
+            ],
+            "allOf": [
+                {
+                    "if": {
+                        "properties": {
+                            "columnType": {
+                                "const": "integer"
+                            },
+                            "status": {
+                                "const": "Complete"
+                            }
```

Comment:
Is the `"then"` part the same irrespective of the `columnType`?
If so, maybe this section could be simplified to:

```
               {
                    "if": {
                        "properties": {
                            "status": {
                                "const": "Complete"
                            }
                        }
                    },
                    "then": {
                        "properties": {
                            "metrics": {
                                "type": "array",
                                "allOf": [
                                    {
                                        "contains": {
                                            "$ref": "#/definitions/exploration_info"
                                        }
                                    }
                                ]
                            }
                        }
                    }
                },
```

Reply:
although I see the started varying a bit more in a later commit... and also cleaned up as part of the yaml transition!